docs/README.md

```diff
@@ -0,0 +1,47 @@
+# Documentation For Developers
+
+Here are some documents important for TEAMMATES developers.
+
+## Main documents
+
+* [**Specification**](https://docs.google.com/document/d/1hjQQHYM3YId0EUSrGnJWG5AeFpDD_G7xg_d--7jg3vU/pub?embedded=true) (short name: **Spec**): Product specification, written from the developers point of view. The user's point of view is [here](http://teammatesv4.appspot.com/features.jsp).
+* The project [**vision, challenges, and principles**](overview.md) from a developer's perspective.
+* Software [**Design**](design.md) - Architecture, APIs, etc.
+* How to [**set up the developer environment**](settingUp.md).
+* The individual development [**process**](development.md) - how to work with your own local copy of TEAMMATES.
+* The team development [**workflow**](process.md).
```

Comment:
Coding/testing instructions might have a name clash with [this document](https://docs.google.com/document/d/14EFJzdhp10qQ9iZ-FwsS1FAf42voe4exrlbefo_zYaU/pub?embedded=true) and [this document](https://docs.google.com/document/d/1aK-1ubIA59fbNsEujwvWKqx7itwLPTD6mzFo4F_oEoI/pub?embedded=true).